anyway/widgets/all_locations_widgets/most_severe_accidents_table_widget.py

```diff
@@ -1,174 +1,169 @@
+        super().__init__(request_params, type(self).name)
+        self.rank = 2
+        self.information = "Most recent fatal and severe accidents, ordered by date. Up to 10 accidents are presented."
+
+    def generate_items(self) -> None:
+        # noinspection PyUnresolvedReferences
+        self.items = MostSevereAccidentsTableWidget.prepare_table(
+            self.request_params.location_info,
+            self.request_params.start_time,
+            self.request_params.end_time,
+            self.request_params.resolution,
+        )
+
+    @staticmethod
+    def prepare_table(
+        location_info, start_time, end_time, resolution: BE_CONST.ResolutionCategories
+    ):
+        entities = (
+            "id",
+            "provider_code",
+            "accident_timestamp",
+            "accident_type",
+            "accident_year",
+            "accident_severity",
+        )
+        accidents = get_most_severe_accidents_with_entities(
+            table_obj=AccidentMarkerView,
+            filters=location_info,
+            entities=entities,
+            start_time=start_time,
+            end_time=end_time,
+            resolution=resolution,
+        )
+        # Add casualties
+        for accident in accidents:
+            accident["type"] = AccidentType(accident["accident_type"]).get_label()
+            dt = accident["accident_timestamp"].to_pydatetime()
+            accident["date"] = dt.strftime("%d/%m/%y")
+            accident["hour"] = dt.strftime("%H:%M")
+            accident["killed_count"] = get_casualties_count_in_accident(
+                accident["id"], accident["provider_code"], 1, accident["accident_year"]
+            )
+            accident["severe_injured_count"] = get_casualties_count_in_accident(
+                accident["id"], accident["provider_code"], 2, accident["accident_year"]
+            )
+            accident["light_injured_count"] = get_casualties_count_in_accident(
+                accident["id"], accident["provider_code"], 3, accident["accident_year"]
+            )
+            # TODO: remove injured_count after FE adaptation to light and severe counts
+            accident["injured_count"] = (
+                accident["severe_injured_count"] + accident["light_injured_count"]
+            )
+            del (
+                accident["accident_timestamp"],
+                accident["accident_type"],
+                accident["id"],
+                accident["provider_code"],
+            )
+            accident["accident_severity"] = AccidentSeverity(
+                accident["accident_severity"]
+            ).get_label()
+        return accidents
+
+    @staticmethod
+    def localize_items(request_params: RequestParams, items: Dict) -> Dict:
+        if request_params.lang != "en":
+            for item in items["data"]["items"]:
+                try:
+                    item["accident_severity"] = _(item["accident_severity"])
+                    item["type"] = _(item["type"])
```

Comment:
Can there be an `item["type"]` with out `item["accident_severity"]`?

Reply:
No, both are keys in the `dict` elements in the list `items["data"]["items"]`. I do not understand, what is the issue?

Reply:
If there is a case where only `item["type"]` can be present then we can get an KeyError on `item["accident_severity"]` and not translating `item["type"]`.
But it's not relevant